lib/url_parser.rb

```diff
@@ -30,12 +30,13 @@ def self.parse_to_full_user_url(url)
   def self.try_all(url)
     GithubURLParser.parse_to_full_url(url) ||
     GitlabURLParser.parse_to_full_url(url) ||
-    BitbucketURLParser.parse_to_full_url(url)
+    BitbucketURLParser.parse_to_full_url(url) ||
+    ApacheSvnUrlParser.parse_to_full_url(url)
   end
 
   def parse_to_full_url
     path = parse
-    return nil unless path.present?
```

Comment:
I was getting a method not found error when calling `present?` when running in a local pry session because we don't have active_support here. I'm wondering if this worked before because it was getting called from within a rails environment and not on its own.

Reply:
That's exactly it. All of this code was ripped out of Libraries and made into a gem so that it was re-usable. I thought I got all of the active support patches, but I guess I missed this one. I haven't seen any bugsnags related to this, so I guess that code path is not used.